charts/argo-rollouts/ci/enable-dashboard-values.yaml

```diff
@@ -4,3 +4,5 @@ installCRDs: false
 
 dashboard:
   enabled: true
+  ingress:
+    enabled: true
```

Comment:
Since it's bundled with dashboard, I just added flag in existing test values.yaml. 🙈